### PR TITLE
[AAE-7639] Added overrides for GWT Terms and ASM

### PR DESCRIPTION
**Pull request by andrea-ligios**

### CODE REVIEW

File: override-THIRD-PARTY.properties [hunk 1]
```diff
@@ -214,6 +216,8 @@ org.latencyutils--LatencyUtils--2.0.3=BSD-2-Clause
 org.openpnp--opencv--3.2.0-1=BSD-3-Clause
 # https://repo1.maven.org/maven2/org/osgi/org.osgi.core/4.2.0/org.osgi.core-4.2.0.jar
 org.osgi--org.osgi.core--4.2.0=Apache-2.0
+# https://gitlab.ow2.org/asm/asm/-/blob/ASM_5_0_4/LICENSE.txt
+org.ow2.asm--asm--5.0.4
```
Comment (tpage-alfresco) [inline override-THIRD-PARTY.properties:220]: This is missing the license name. The tests passed - oops.

**Review by tpage-alfresco — COMMENTED** [review]
I was expecting you would just merge GWT Terms with Apache-2.0.  I think that for the purposes of this project (due to the way java imports dependencies) then the two licenses are equivalent.

File: override-THIRD-PARTY.properties (hunk 1 above)
Comment (andrea-ligios) [reply override-THIRD-PARTY.properties]: Fixed

**Review by MichalFidor — APPROVED** [review]
Lgtm!

**Pull request merged**